go.mod

```diff
@@ -128,9 +122,12 @@ require (
 replace (
 	bazil.org/fuse => bazil.org/fuse v0.0.0-20200117225306-7b5117fecadc // specific version for iOS building
 	github.com/agl/ed25519 => github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412 // latest commit before the author shutdown the repo; see https://github.com/golang/go/issues/20504
-	github.com/ipfs/go-ipfs => github.com/Jorropo/go-ipfs v0.4.20-0.20201127133049-9632069f4448 // temporary, see https://github.com/ipfs/go-ipfs/issues/7791
-	github.com/libp2p/go-libp2p-rendezvous => github.com/berty/go-libp2p-rendezvous v0.0.0-20201028141428-5b2e7e8ff19a // use berty fork of go-libp2p-rendezvous
-	github.com/libp2p/go-libp2p-swarm => github.com/Jorropo/go-libp2p-swarm v0.4.2 // temporary, see https://github.com/libp2p/go-libp2p-swarm/pull/227
+
+	github.com/libp2p/go-libp2p-core => github.com/libp2p/go-libp2p-core v0.9.0
```

Comment:
Is it not enough to bump the version in the `require` block? Why keep it the in the `replace`?

Reply:
I need to force the version of go-ipfs-core to 0.9.0 otherwise it will bump to a 0.10.0, and the project wont build, this include some other upgrade on some repo.
we should remove this after the update on go-ipfs on 0.10.0